init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+:CheckOS
+IF "%PROCESSOR_ARCHITECTURE%"=="x86" (set bit=x86) else (set bit=x64)
```

Comment:
So we use the 32-bit CLI to build the 32-bit CLI.  The same reason we have stage1 and stage2 - test coverage.